src/use-swr.ts

```diff
@@ -24,7 +24,7 @@ import {
   updaterInterface
 } from './types'
 
-const IS_SERVER = typeof window === 'undefined'
+const IS_SERVER = typeof window === 'undefined' || (typeof Deno !== 'undefined' && Deno !== null && !!Deno.version && !!Deno.version.deno)
```

Comment:
```suggestion
// @ts-ignore
const IS_SERVER = typeof window === 'undefined' || !!(typeof Deno !== 'undefined' && Deno && Deno.version && Deno.version.deno)
```

It's fine to ignore the TS error here, otherwise we need an extra TS plugin.